src/test/java/org/fundacionjala/sfdc/stepdefinition/AccountSteps.java

```diff
@@ -0,0 +1,141 @@
+ */
+public class AccountSteps {
+    private WebDriver webDriver;
+
+    private SalesForceAccounts salesForceAccounts;
+    private Properties propertiesManager;
+
+    /**
+     * Given step.
+     */
+    @Given("^Go to accounts section$")
+    public void goToAccountsSection() {
+        System.out.println("This Step goes to accounts section");
+        webDriver = DriverManager.getInstance().getNavigator();
+        SalesForceMainTab salesForceMainTab;
+        salesForceMainTab = new SalesForceMainTab(webDriver);
+        salesForceMainTab.displayOptions();
+        salesForceMainTab.goToAccounts();
+    }
+
+    /**
+     * When step.
+     */
+    @When("^I press new account button$")
+    public void iPressNewAccountButton() {
+        System.out.println("This Step presses new account button");
+        webDriver = DriverManager.getInstance().getNavigator();
+        salesForceAccounts = new SalesForceAccounts(webDriver);
+    }
+
+    /**
+     * Then step.
+     */
+    @Then("^new account form is displayed$")
+    public void newAccountFormIsDisplayed() {
+        System.out.println("This Step displays new accounts form");
+        webDriver = DriverManager.getInstance().getNavigator();
+        salesForceAccounts = new SalesForceAccounts(webDriver);
+        Assert.assertTrue(salesForceAccounts.isNewAccountButtonDisplayed());
+    }
+
+    /**
+     * Then step.
+     */
+    @Then("^I fill the account name field$")
+    public void iFillTheAccountNameField() {
+        System.out.println("This Step fills account name field");
+        webDriver = DriverManager.getInstance().getNavigator();
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        salesForceAccounts = new SalesForceAccounts(webDriver);
+        salesForceAccounts.createNewAccount(propertiesManager.getProperty("accountName"));
+    }
+
+    /**
+     * And step.
+     */
+    @And("^press the save button$")
+    public void pressTheSaveButton() {
+        System.out.println("This Step saves new account");
+        webDriver = DriverManager.getInstance().getNavigator();
+        salesForceAccounts = new SalesForceAccounts(webDriver);
+        salesForceAccounts.clickSaveNewAccountButton(salesForceAccounts.getSaveNewAccountButton());
+    }
+
+    /**
+     * Then step.
+     */
+    @Then("^a new account is created$")
+    public void aNewAccountIsCreated() {
+        webDriver = DriverManager.getInstance().getNavigator();
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        salesForceAccounts = new SalesForceAccounts(webDriver);
+        System.out.println("This Step tests new account creation:" + salesForceAccounts.newAccountSavedName());
+        Assert.assertEquals(salesForceAccounts.newAccountSavedName(), propertiesManager.getProperty("accountName"));
+    }
+
+    /**
+     * After, to close browser after the tests are executed.
+     */
+    /*@After
+    public void closeBrowser() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)